docs/content/en/docs/getting-started.md

```diff
@@ -60,11 +60,11 @@ These things can be found in our Getting Started repository
 which can be checked out as follows:
 
 ```shell
-git clone https://github.com/keptn-sandbox/lifecycle-toolkit-examples.git
-cd lifecycle-toolkit-examples
+git clone https://github.com/keptn/lifecycle-toolkit
+cd lifecycle-toolkit/examples
 ```
 
```

Comment:
```suggestion
```
markdown lint doesn't like to blank lines in a row.  I think this will fix it.  You can also run `make markdownlint-fix` to automatically fix most of the markdown lint errors.

Reply:
after a code fence, it requires 2 new lines 😅 
I know, it takes a while to get use to it

Reply:
Ah, that explains why markdownlint passed this with those two spaces.  Aargh!  Although I am pretty sure that I have source with only a single blank line after a code fence that is being passed.  Ah well, we'll get used to it in time.